rs/bitcoin/kyt/src/dashboard/tests.rs

```diff
@@ -0,0 +1,201 @@
+use crate::dashboard::tests::assertions::DashboardAssert;
+use crate::dashboard::{filters, DashboardTemplate, Fetched, Status};
+use crate::state::Timestamp;
+use crate::BtcNetwork;
+use bitcoin::Address;
+use ic_btc_interface::Txid;
+use std::str::FromStr;
+
+fn mock_txid(v: u8) -> Txid {
+    Txid::from([v; 32])
+}
+
+#[test]
+fn should_display_metadata() {
+    let btc_network = BtcNetwork::Mainnet;
+    let outcall_capacity = 50;
+    let dashboard = DashboardTemplate {
+        btc_network,
+        outcall_capacity,
+        fetch_tx_status: Vec::new(),
+    };
+
+    DashboardAssert::assert_that(dashboard)
+        .has_btc_network_in_title(btc_network)
+        .has_outcall_capacity(outcall_capacity)
+        .has_no_status();
+}
+
+fn parse_address(addresses: &[&str]) -> Vec<Option<Address>> {
+    addresses
+        .iter()
+        .map(|s| Some(Address::from_str(s).unwrap().assume_checked()))
+        .collect()
+}
+
+#[test]
+fn should_display_statuses() {
```

Comment:
nit: Maybe add a unit test that ok addresses are green, while blocked addresses are red?

Reply:
Good idea! Will add such a test

Reply:
Added a test